### PR TITLE
NoFall improvements & no centerPlayer overshoot

### PR DESCRIPTION
**Pull request by ThebestkillerTBK**

### CODE REVIEW

File: src/main/java/meteordevelopment/meteorclient/utils/player/PlayerUtils.java [hunk 1]
```diff
@@ -107,6 +107,9 @@ public static void centerPlayer() {
         double x = MathHelper.floor(mc.player.getX()) + 0.5;
         double z = MathHelper.floor(mc.player.getZ()) + 0.5;
         mc.player.setPosition(x, mc.player.getY(), z);
+        Vec3d pVel = mc.player.getVelocity();
+        if (pVel.x < 0.1 || pVel.z < 0.1) return;
+        pVel.multiply(0.5);
         mc.player.networkHandler.sendPacket(new PlayerMoveC2SPacket.PositionAndOnGround(mc.player.getX(), mc.player.getY(), mc.player.getZ(), mc.player.isOnGround()));
```
Comment (MineGame159) [inline src/main/java/meteordevelopment/meteorclient/utils/player/PlayerUtils.java:113]: Shouldn't it send the packet regardless if it modifiyed the velocity?

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/NoFall.java [hunk 2]
```diff
@@ -185,8 +174,6 @@ else if (mode.get() == Mode.Place) {
     }
 
     private void useItem(FindItemResult item, boolean placedWater, BlockPos blockPos, boolean interactItem) {
-        if (!item.found()) return;
```
Comment (MineGame159) [inline src/main/java/meteordevelopment/meteorclient/systems/modules/movement/NoFall.java:188]: Why?

**Review by MineGame159 — CHANGES_REQUESTED** [review]

File: src/main/java/meteordevelopment/meteorclient/systems/modules/movement/NoFall.java (hunk 2 above)
Comment (ThebestkillerTBK) [reply src/main/java/meteordevelopment/meteorclient/systems/modules/movement/NoFall.java:188]: The code has done item search in the previous code.

File: src/main/java/meteordevelopment/meteorclient/utils/player/PlayerUtils.java (hunk 1 above)
Comment (ThebestkillerTBK) [reply src/main/java/meteordevelopment/meteorclient/utils/player/PlayerUtils.java:113]: If you got moved the velocity wont change and it always overshoot if you use place mode in nofall

**Pull request closed without merging**